build/deb_rpm/build-scripts/pack.sh

```diff
@@ -323,7 +331,7 @@ main(){
 	: ${DEB_TEST_IMAGE:="${DEB_BUILD_IMAGE}"}
 	: ${RPM_TEST_IMAGE:="${RPM_BUILD_IMAGE}"}
 	: ${JFROG_CLI_RELEASE_VERSION:="1"}
-	: ${RPM_SIGN_PASSPHRASE:=""}
+	: ${RPM_SIGN_PASSPHRASE:=$(cat $RPM_SIGN_PASSPHRASE_FILE)}
```

Comment:
I'm not sure it is necessary, but I think `"$(cat $RPM_SIGN_PASSPHRASE_FILE)"` may be useful.